scripts/check_skill_pairs.py

```
@@ -0,0 +1,70 @@
+    (".gemini/skills/mcp-activation/SKILL.md", ".github/skills/mcp-activation/SKILL.md"),
+    (".gemini/skills/project-navigation/SKILL.md", ".github/skills/project-navigation/SKILL.md"),
+    (".gemini/skills/pr-analysis/SKILL.md", ".github/skills/pr-analysis/SKILL.md"),
+    (".gemini/skills/logging-standards/SKILL.md", ".github/skills/logging-standards/SKILL.md"),
+    (".gemini/skills/devcontainer-management/SKILL.md", ".github/skills/devcontainer-services/SKILL.md"),
+    (".gemini/skills/devcontainer-management/SKILL.md", ".github/skills/devcontainer-setup/SKILL.md"),
+    (".gemini/skills/devcontainer-management/SKILL.md", ".github/skills/devcontainer-configs/SKILL.md"),
+]
+
+
+def changed_files(base_ref):
+    result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)